### ISSUE TITLE
HyperlinkUnderline skinparam is broken.

### ISSUE THREAD

**Issue by tamas-p** [plantuml/plantuml#1241]
HyperlinkUnderline skinparam seems to be broken since V1.2022.2 (last working version is V1.2022.1).

Example plantuml file:
```plantuml
@startuml
skinparam HyperlinkUnderline false
state "[[S1]]" as S1
@enduml
```

With V1.2022.1 it is
![test_ok](https://user-images.githubusercontent.com/10595775/210099423-3c69b857-735f-4ed8-888f-9d46f982e14e.png)

Since V1.2022.2 it is
![test_nok](https://user-images.githubusercontent.com/10595775/210099427-a2ef0ab8-707f-493f-92e1-b29353f25735.png)

Also, see it reported here: https://forum.plantuml.net/2866/decreasing-visibility-of-links?show=15826#c15826

**Comment by arnaudroques** [comment]
Thanks for the feedback, we are going to fix this.
As temporary workaround, [you can have](https://www.plantuml.com/plantuml/umla/SoWkIImgAStDuR8fBgdCIRNZIir9pKtDAr6evb9GoAWiI2tAoSpB3irB0JDIGp8oaxFpKekB5GouQhbizA6QWAQKf2eeHKS76yR6AYaa5Yi46t8vfEQb04q60000):

```
@startuml
<style>
element {
  hyperlinkUnderlineThickness 0
}
</style>
state "[[S1]]" as S1
@enduml
```

Hope this helps!

**Comment by arnaudroques** [comment]
This is fixed in V1.2023.0

**Comment by The-Lum** [comment]
Issue Review _(with Happy resolution 2023: see [Issue tracking](https://forum.plantuml.net/17257/issue-tracking) [FYI @BenceSzalai])_: 
- This issue can be closed.

_Thanks a lot everybody for your contribution._
